docker-entrypoint.sh

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/env bash
```

Comment:
Shall we add a permalink at the top to clearly show where we git it from?

Reply:
`s/git/get/`

Reply:
Absolutely- good idea